code/modules/borer/borer_chemicals.dm

```diff
@@ -0,0 +1,103 @@
+/datum/borer_chem
```

Comment:
make sure they have CM-based descs and names - 'anti toxin' stands out

Reply:
Fiddly mainly because the chem name in the menu is the ID it uses, it can be changed but would require some reformatting on what it's checking. Would be nice though, so definitely on the list to do.